src/main/java/frc/robot/cv/CVDriveCommand.java

```diff
@@ -0,0 +1,126 @@
+package frc.robot.cv;
+
+import edu.wpi.first.wpilibj2.command.CommandBase;
+import edu.wpi.first.wpilibj2.command.SequentialCommandGroup;
+import frc.robot.commands.*;
+import frc.robot.subsystems.DriveSubsystem;
+import frc.robot.Robot;
+
+public class CVDriveCommand extends SequentialCommandGroup {
+
+    DriveSubsystem driveSubsystem;
+    CVData cvData;
+    SequentialCommandGroup path;
+    private boolean isFinished = false;
+    SequentialCommandGroup RedPathA;
+    SequentialCommandGroup RedPathB;
+    SequentialCommandGroup BluePathA ;
+    SequentialCommandGroup BluePathB;
+  
+
+
+    public CVDriveCommand(DriveSubsystem subsystem) {
+
+        driveSubsystem = subsystem;
+        addRequirements(subsystem);
+
+        RedPathA = 
+        new SequentialCommandGroup (
+            new StraightDrive(driveSubsystem, 60, 0.6),
+            new TurnUsingGyro(driveSubsystem, -26.6),
+            new StraightDrive(driveSubsystem, 67.1, 0.6),
+            new TurnUsingGyro(driveSubsystem, 71.6),
+            new StraightDrive(driveSubsystem, 94.9, 0.6),
+            new TurnUsingGyro(driveSubsystem, 0)
+        );
+        RedPathB = 
+        new SequentialCommandGroup (
+            new StraightDrive(driveSubsystem, 60, 0.6),
+            new TurnUsingGyro(driveSubsystem, -45),
+            new StraightDrive(driveSubsystem, 84.9, 0.6),
+            new TurnUsingGyro(driveSubsystem, 45),
+            new StraightDrive(driveSubsystem, 84.9, 0.6),
+            new TurnUsingGyro(driveSubsystem, 0)
+        );
+        BluePathA = 
+        new SequentialCommandGroup (
+            new StraightDrive(driveSubsystem, 150.0, 0.6),
+            new TurnUsingGyro(driveSubsystem, 71.6),
+            new StraightDrive(driveSubsystem, 94.9, 0.6),
+            new TurnUsingGyro(driveSubsystem, -26.6),
+            new StraightDrive(driveSubsystem, 67.1, 0.6),
+            new TurnUsingGyro(driveSubsystem, 0)
+        );
+        //new TurnUsingGyro(driveSubsystem, -21.8),
+
+        BluePathB = 
+        new SequentialCommandGroup (
+            new StraightDrive(driveSubsystem, 120.0, 0.6),
+            new TurnUsingGyro(driveSubsystem, 45),
+            new StraightDrive(driveSubsystem, 84.9, 0.6),
+            new TurnUsingGyro(driveSubsystem, -45),
+            new StraightDrive(driveSubsystem, 84.9, 0.6),
+            new TurnUsingGyro(driveSubsystem, 0)
+        );
+        //new TurnUsingGyro(driveSubsystem, -21.8),
+        
+    }
+
+    @Override
+    public void initialize() { 
+        cvData = Robot.getCVData();
+        System.out.println("Path: " + (cvData.isRedPath() ? "Red " : "Blue ") + (cvData.isPathA() ? "A" : "B"));
+       if(cvData.isRedPath() && cvData.isPathA()) {
```

Comment:
We can move all of this into autonomousInit, avoiding this weird double nested SequentialCommandGroup business. in autonomousInit, we are already running jetson.updateVision(). We can then get a decision on the path from jetson.getCVData(), then do something like 
```
if(path==RED_A) {
RedPathA.schedule()
} else if (path==RED_B) {
RedPathB.schedule()
}
etc.
```